src/components/event/Event.js

```diff
@@ -23,18 +23,19 @@ export const Event = ({ event, setViewingEvent, setShowingEventForm, deleteEvent
       <Modal onClose={() => setViewingEvent(null)} title={`${event.name} (${event.park.name})`} className="eventModal">
         <p>From <b>{event.start_date}</b> to <b>{event.end_date}</b></p>
         <p>{event.description}</p>
+        <FavoriteBtn resource={"events"} resource_id={Number(event.id)}/>
         <>
         { isStaff? <>
-        <button href="javascript:;" onClick={() => {
+        <button className="regButton" href="javascript:;" onClick={() => {
```

Comment:
`href` is not a valid attribute on a `<button>` element.

Reply:
taken off!